CO2_Gadget_EINK.h

```diff
@@ -232,6 +234,15 @@ void setElementLocations() {
     elementPosition.smallFontDigitsHeight = tbh;
 }
 
+bool showBatteryIcon(int32_t posX, int32_t posY, bool forceRedraw);
+bool showCO2(uint16_t co2, int32_t posX, int32_t posY, bool forceRedraw);
+bool showHumidity(float hum, int32_t posX, int32_t posY, bool forceRedraw);
+bool showTemperature(float temp, int32_t posX, int32_t posY, bool forceRedraw);
+bool showBLEIcon(int32_t posX, int32_t posY, bool forceRedraw);
+bool showWiFiIcon(int32_t posX, int32_t posY, bool forceRedraw);
+bool showMQTTIcon(int32_t posX, int32_t posY, bool forceRedraw);
+bool showEspNowIcon(int32_t posX, int32_t posY, bool forceRedraw);
+
```

Comment:
Let's discuss this...
A different interface to the rest of the displays means more difficulty to integrate in just one class with a unique interface definition. I'm not sure it's needed (see other comments).

Reply:
This is necessary only if partial refresh of only a small part of displys is implemented, because we need to know if a value (CO2, temp, hum., batt. or icons) has changed from previous displayed values.

If we only implement a full partial display (display.setPartialWindow(0, 0, display.width(), display.height());) this is not necessary.

Another way is define global variables to know if this part of screen must be refreshed. If you think it's better like this, I can do it.